webpack.config.js

```diff
@@ -30,20 +30,20 @@ module.exports = {
         modules: [
           pathutil.resource('app/components')
           , 'web_modules'
-          , 'bower_components'
           , 'node_modules'
         ]
-        , descriptionFiles: ['package.json', 'bower.json']
+        , descriptionFiles: ['package.json']
         , moduleExtensions: ['-loader']
         , extensions: ['.js', '.json']
         , enforceModuleExtension: false
-        , alias: {
-            'angular-bootstrap': 'angular-bootstrap/ui-bootstrap-tpls'
-            , localforage: 'localforage/dist/localforage.js'
-            , 'socket.io': 'socket.io-client'
-            , stats: 'stats.js/src/Stats.js'
-            , 'underscore.string': 'underscore.string/index'
-        }
+      , alias: {
+        '@devicefarmer/adbkit': './adbkit'
+        , 'angular-bootstrap': 'angular-bootstrap/ui-bootstrap-tpls'
+        , localforage: 'localforage/dist/localforage.js'
+        , 'socket.io': 'socket.io-client'
+        , stats: 'stats.js/src/Stats.js'
+        , 'underscore.string': 'underscore.string/index'
+      }
```

Comment:
add 2 spaces to the `alias `section to align it correctly

Reply:
fixed